bin/main.cpp

```diff
@@ -13,7 +13,7 @@ int main(int argc, char** argv) {
 	game->states().push(std::make_shared<StateMenu>(game));
 
 	// game update loop
-	auto loop_dt = std::chrono::nanoseconds(long(std::round(glob::dt * 10E9)));
+	auto loop_dt = std::chrono::nanoseconds(long(std::round(glob::dt * 10E8))); // ~16 milliseconds
```

Comment:
I increased our game to actually 60FPS, it was 6FPS

Reply:
lol! oopsie.